custom_components/waste_collection_schedule/waste_collection_schedule/source/obdach_at.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Marktgemeinde Obdach"
+DESCRIPTION = "Source for Marktgeneinde Obdach, AT"
+URL = "https://www.obdach.gv.at/"
+TEST_CASES = {"TestSource": {}, "IgnoredArgument": {"_": ""}}
+ICON_MAP = {
+    "Biomüll": "mdi:food",
+    "Altstoffsammelzentrum": "mdi:newspaper",
+    "Gelber Sack/Tonne": "mdi:recycle-variant",
+    "Restmüll Abfuhrbereich 1": "mdi:trash-can",
+    "Restmüll Abfuhrbereich 2": "mdi:trash-can",
+}
+
+
+class Source:
+    def __init__(self, _=None):
```

Comment:
You don't need the dummy argument, it just makes configuration more confusing  (The HA config GUI shows this field)